doc/manual/manual.tex

```diff
@@ -9722,6 +9722,51 @@ \subsubsection{Thin shell gravity benchmark}
 The accuracy of the calculations increases with $I$ but so does the time spent in the postprocessor: for $I\in\{-1,0,1,2\}$  this time was about 18, 132, 440 and 1040~\si{s} respectively. This is easily explained when one realizes that increasing $I$ from 0 to 1 means that the number of GLQ points per cell goes from $2^3=8$ to $3^3=27$, i.e. a $3.375$ increase in operations for the same number of cells and measurement points. The time spent in the postprocessor increases by a similar factor ($440/132\sim 3.4$).
 The accuracy obtained with lower $I$ values increases with increasing anomaly depth (or increasing distance from the observation point). Note that this benchmark has uniform density so that the projection of the density from the nodes onto the quadrature points is exact and it does not introduce any smoothing of data which might occur in the presence of density discontinuities (e.g. air-water, water-crust, moho, etc.) inside a cell. $I=1$ seems to be the best compromise between accuracy (gravity acceleration errors are less than 0.01~mGal for all shells) and compute time for this experiment.
 
+
+
+
+\subsubsection{Gravity field generated by mantle density variations}
+\label{sec:benchmark-mantle-gravity}
+
+\textit{This section was contributed by C. Thieulot and L. Jeanniot.}
+
+The gravity postprocessor has been benchmarked in Section~\ref{sec:benchmark-thin-shell-gravity} and \ref{sec:benchmark-thick-shell-gravity}.
```

Comment:
The thick shell gravity benchmark reference is not found. Is that a different open PR, or do you mean a different section?